### PR TITLE
Promote DownwardAPIHugePages to beta

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Documentation for associated KEP https://github.com/kubernetes/enhancements/pull/2352

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
There was no functional change relative to the API behavior in Alpha versus Beta, so prior docs aside from feature-gate update are fine.  See: https://github.com/kubernetes/website/pull/24904

**Comment by reylejano** [comment]
@derekwaynecarr Is this also related to kubernetes/enhancements#2053?

**Comment by sftim** [comment]
https://k8s.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/ is written in a way that doesn't need changes here.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ different Kubernetes components.
 | `DevicePlugins` | `true` | Beta | 1.10 | |
 | `DisableAcceleratorUsageMetrics` | `false` | Alpha | 1.19 | 1.19 |
 | `DisableAcceleratorUsageMetrics` | `true` | Beta | 1.20 | |
-| `DownwardAPIHugePages` | `false` | Alpha | 1.20 | |
+| `DownwardAPIHugePages` | `false` | Beta | 1.21 | |
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:102]: 👋 @derekwaynecarr .
Would you leave the `Alpha` item in the table and add a value in the last column (add the last version that this feature was Alpha).
```
|`DownwardAPIHugePages` | `false` | Alpha | 1.20 | 1.20 |
```

Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:102]: It'd be like this:
```suggestion
| `DownwardAPIHugePages` | `false` | Alpha | 1.20 | 1.20 |
| `DownwardAPIHugePages` | `false` | Beta | 1.21 | |
```
(feel free to commit this via GitHub, fetch, and squash - or fix locally. Either is fine.)

**Comment by derekwaynecarr** [comment]
@sftim made the requested update, thanks for the tip!

**Comment by derekwaynecarr** [comment]
@ChandaniM123 @reylejano I think I addressed the desired updates if you can take a look.

**Pull request merged**